src/main/java/com/ratifire/devrate/service/notification/model/InterviewScheduleNotificationData.java

```diff
@@ -0,0 +1,19 @@
+package com.ratifire.devrate.service.notification.model;
+
+import com.ratifire.devrate.entity.User;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+/**
+ * Data record for interview schedule notification information.
+ */
+public record InterviewScheduleNotificationData(
```

Comment:
do we really need this record? Can we reuse  something similar that already exist?